modos/cli.py

```diff
@@ -142,24 +139,24 @@ def create(
 ):
     """Create a modo interactively or from a file."""
     typer.echo("Creating a digital object.", err=True)
-    # Initialize object's directory
-    if object_directory.exists():
-        raise ValueError(f"Directory already exists: {object_directory}")
 
-    if s3_endpoint:
-        fs = connect_s3(s3_endpoint, {"anon": True})
+    endpoint = ctx.obj.endpoint
+    # Initialize object's directory
+    if endpoint:
+        s3 = list_endpoints(endpoint)["s3"]  # type: ignore
```

Comment:
__todo__: `list_endpoints()` is not defined and `modos create --endpoint` will break.
Maybe worth implementing in `modos.remote`?
Or something like this:
```
s3 = EndpointManager(endpoint).s3
```

Reply:
df7ff1276b01fa1bc1ed72066fd5e215730bc26c :heavy_check_mark: